src/aioswitcher/api/__init__.py

```diff
@@ -704,6 +716,32 @@ def _lookup_key_in_irset(self, key: list):
                 # found a match, with modified list
                 return
 
+    def get_swing_command(self, swing: ThermostatSwing):
```

Comment:
I think some tests might help clarify this function to those of us who don't know the internals of the thermostat swing thing, like me :-)

Reply:
It is covered in the test multiple times as this function is major entry point for controlling Breeze

Reply:
> It is covered in the test multiple times as this function is major entry point for controlling Breeze

can you provide a reference?